fvm/src/kernel/default.rs

```diff
@@ -1,30 +1,24 @@
 use std::collections::BTreeMap;
 use std::convert::{TryFrom, TryInto};
+use std::panic::{self, UnwindSafe};
 
 use anyhow::{anyhow, Context as _};
 use byteorder::{BigEndian, WriteBytesExt};
 use cid::Cid;
-use filecoin_proofs_api::seal::{
-    compute_comm_d, verify_aggregate_seal_commit_proofs, verify_seal as proofs_verify_seal,
-};
-use filecoin_proofs_api::update::verify_empty_sector_update_proof;
-use filecoin_proofs_api::{self as proofs, post, seal, ProverId, PublicReplicaInfo, SectorId};
+use filecoin_proofs_api::{self as proofs, ProverId, PublicReplicaInfo, SectorId};
```

Comment:
I started qualifying these calls because distinguishing between "our functions" and "not our functions" was getting difficult.